acme/src/test/groovy/io/micronaut/acme/AcmeCertRefresherMultiDomainsTaskSpec.groovy

```diff
@@ -52,13 +52,17 @@ class AcmeCertRefresherMultiDomainsTaskSpec extends AcmeBaseSpec {
                 try {
                     conn.connect()
                     Certificate[] certs = conn.getServerCertificates()
-                    certs.length == 1
+                    certs.length == 2
                     def cert = (X509Certificate) certs[0]
                     cert.getIssuerDN().getName().contains("Pebble Intermediate CA")
                     cert.getSubjectDN().getName().contains(EXPECTED_DOMAIN)
                     cert.getSubjectAlternativeNames().size() == 2
                     cert.getSubjectAlternativeNames().collect({d-> d.get(1)}).contains(EXPECTED_DOMAIN)
                     cert.getSubjectAlternativeNames().collect({d-> d.get(1)}).contains(EXPECTED_ACME_DOMAIN)
+
+                    def cert2 = (X509Certificate) certs[1]
+                    cert2.getIssuerDN().getName().contains("Pebble Root CA")
```

Comment:
In groovy you can be more idiomatic by rewriting this:

`cert2.getIssuerDN().getName().contains("Pebble Root CA")`

as: 

`cert2.issuerDN.name.contains("Pebble Root CA")`

Reply:
Grooy-ify things here [075cb6c](https://github.com/micronaut-projects/micronaut-acme/pull/161/commits/075cb6cd314f4d3d933115e453b9cad544a2c4b7)